release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -424,6 +424,11 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-release-notes-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4-19-installation-and-update-remove-terraform-azure-stack-hub_{context}"]
+==== Cluster API replaces Terraform for {azure-short} Stack Hub
+
+In {product-title} {product-version}, the installation program uses the Cluster API instead of Terraform to provision clusters during installer-provisioned infrastructure (IPI) installations on {azure-short} Stack Hub.
```

Comment:
I'm not sure if "IPI" is an official abbreviation we should be using. Maybe check with Ben Scott.